### DISCUSSION TITLE
What is the best practice to personalize the desktop

### DISCUSSION THREAD

**Discussion by jessybreyne** [Manjaro-Sway/manjaro-sway#334] [Q&A] [answered]
What is the best practice to personalize the desktop? (Change colors, widget, city of weather...)

Change directly in admin the code in /etc or in /usr/share?

Can we copy in our home .config? Why not put the config in the home at the installation?
In this case, which configuration is taken in priority, the one in the home or in etc or usr?

Do the configs in /etc or /usr update with system updates?

**Comment by boredland** [comment, marked as answer]
hi there! we tried to make all configs extensible. refer [here](https://github.com/Manjaro-Sway/manjaro-sway/blob/main/SUPPORT.md) for pointers. if you miss a topic, feel free to ask.

**Reply by jessybreyne** [reply]
Hi! I love the work and setup you have done. But I'm sorry, I don't understand this choice of configuration. 
Why not put directly in the home? It seems much easier to configure than having to go see in etc, which file to copy and take all its content to edit it...

It's not more easy to take all our configs in home and git it to save it for the next time?
```
cp /etc/sway/ ~/.config/sway/
```
Same for the content in /usr/share

This may be a stupid question but I'd like to see how to configure a lot of stuff in the cleanest way possible 😄

**Reply by boredland** [reply]
the general assumption is, that you will only configure a tiny fraction of it - e.g. some colors or adding a module to waybar. if you want to build everything from scratch, which you certainly can, why would you not just overwrite `.config/sway/config` entirely? 

The reason we have it like that is that basic assumption - convention extensible by configuration. This has one great benefit: we can update it without causing friction (scripts, modules etc). We couldn't do that, if everything was in $HOME.
